code/game/gamemodes/meteor/meteors.dm

```diff
@@ -142,6 +143,9 @@
 					!istype(A,/obj/machinery/field_generator) && \
 					prob(detonation_chance))
 					explosion(loc, power, power + power_step, power + power_step * 2, power + power_step * 3, 0)
+					msg_admin_attack("Meteor exploded at coords <[x],[y],[z]>")

```

Comment:
Please change all of these `<[x],[y],[z]>` references to this: `(<a href='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)`